driver/helpers.c

```diff
@@ -0,0 +1,23 @@
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+#ifdef _WIN32
+int setenv(const char *name, const char *value, const int overwrite) {
+    if (overwrite) goto replace;
+    size_t n = 0;
+    const int errcode = getenv_s(&n, NULL, 0, name);
```

Comment:
It may be better to use `getenv` and `_putenv` to get better consistency since POSIX version of getenv and putenv is not thread-safe.